cli/explain/compactgraph/testdata/generate.go

```diff
@@ -798,6 +838,20 @@ sh_binary(
 `,
 			bazelVersions: []string{"8.0.0"},
 		},
+		{
+			name:     "tool_runfiles_empty_files_identical",
+			baseline: ToolRunfilesEmptyFilesProject,
+			changes: `
+-- pkg/lib/__init__.py --
+`,
+			bazelVersions: []string{"8.0.0"},
+		},
+		{
+			name:          "tool_runfiles_empty_files_differ",
+			baseline:      ToolRunfilesEmptyFilesProject,
+			changedArgs:   []string{"--incompatible_default_to_explicit_init_py"},
```

Comment:
I thought these python specific flag have all gone away with rules_python by now 💀

Reply:
This particular functionality can't be expressed in Starlark (yet and likely never), so it's accessible only from `rules_python` but still implemented in the Bazel core.